.travis.yml

```diff
@@ -37,8 +37,8 @@ addons:
     - yarn
 
 rvm:
-  - 2.3.4
-  - 2.4.2
+  - 2.3.6
```

Comment:
Does anyone think we still need to test on the 2.3 branch?

Reply:
Ruby 2.5 will be released on the 24th of this month. I want to add CI 2.5 and drop 2.3.